### DISCUSSION TITLE
Advanced Sub Station Alpha settings don't burn correctly

### DISCUSSION THREAD

**Discussion by flgal** [SubtitleEdit/subtitleedit#6550] [Q&A]
v.3.6.10 
using subtitle format Advanced Sub Station Alpha "A"
video player mpv
profile: Netflix

I set the A setting to a font size of 33 (which looks like I want it in the player) but when I burn the video, I have to set the font size to 200 so that it looks the same as when I viewed in mpv player.

Same with changing the margin settings. I looks fine in player but wont' change the margin when I burn (burning in 3840 x 2160)

Any suggestions?
Thanks

UPDATE:No need for reply.

 I found the solution. Found ASSA tools on right click and was able to change position of the lines - so that took care of the  margins problem.

As for the font problem, it seemed to fix itself and was able to set  font to 150 in the properties box and have it look like I wanted in the player. Then burned, and it burned just fine so not sure what I was doing before. 

Thanks again for this great software..it rocks!

**Comment by niksedk** [comment]
The default video resolution for ASSA files is 384x288, so it's normally a good idea to set the "PlayResX" and "PlayResY" in the [Script Info] header to the actual video resolution.

**Comment by GrampaWildWilly** [comment]
Surely there's a way to ban this spammer.

**Comment by GrampaWildWilly** [comment]
Excuse me.  Spammers, plural.  There appear to be two . . . so far.
